rustls/src/server/tls13.rs

```diff
@@ -1065,9 +1065,10 @@ impl State<ServerConnectionData> for ExpectEarlyData {
                     send_ticket: self.send_ticket,
                 }))
             }
-            payload => Err(inappropriate_message(
+            payload => Err(inappropriate_handshake_message(
```

Comment:
This would call `inappropriate_handshake_message()` even if the `MessagePayload` is an `Alert` or `ChangeCipherSpec`, which seems a little confusing.

Reply:
> This would call `inappropriate_handshake_message()` even if the `MessagePayload` is an `Alert` or `ChangeCipherSpec`, which seems confusing.

That's already the case throughout the codebase. The new `inappropriate_handshake_message` handles those kinds of messages appropriately.